rosdistro: 'noetic

  '
roslaunch:
  uris:
    host_emreubuntu__46849: http://emreubuntu:46849/
rosversion: '1.15.14

  '
run_id: c8800aa2-b95d-11ec-8f2a-17dba42a9e32
goalX_r1: 8.0
goalY_r1: 5.0
goalX_r2: 1.0
goalY_r2: 2.0

